pkgs/by-name/py/pybind11-abseil/package.nix

```diff
@@ -0,0 +1,38 @@
+{
+  lib,
+  stdenv,
+  fetchFromGitHub,
+  cmake,
+  abseil-cpp,
+  python3,
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "pybind11-abseil";
+  version = "202402.0";
+
+  src = fetchFromGitHub {
+    owner = "pybind";
+    repo = "pybind11_abseil";
+    tag = "v${finalAttrs.version}";
+    hash = "sha256-hFVuGzEFqAEm2p2RmfhFtLB6qOqNuVNcwcLh8dIWi0k=";
+  };
+
+  nativeBuildInputs = [ cmake ];
+
+  buildInputs = [ python3.pkgs.pybind11 ];
+
+  cmakeFlags = [
+    (lib.cmakeBool "FETCHCONTENT_FULLY_DISCONNECTED" true)
+    (lib.cmakeFeature "FETCHCONTENT_TRY_FIND_PACKAGE_MODE" "ALWAYS")
+    (lib.cmakeFeature "FETCHCONTENT_SOURCE_DIR_ABSEIL-CPP" "${abseil-cpp.src}")
+    (lib.cmakeFeature "CMAKE_INSTALL_PYDIR" "${placeholder "out"}/${python3.sitePackages}")
+  ];
+
```

Comment:
`passthru.updateScript = gitUpdater { rev-prefix = "v"; };`

Reply:
If it is a simple updateScript, r-ryantm will update without it.